internal/pkg/template/templates/environment/partials/ar-vpc-connector.yml

```diff
@@ -0,0 +1,48 @@
+AppRunnerVpcEndpointSecurityGroup:
+  Metadata:
+    'aws:copilot:description': 'A security group to services in this environment talk to App Runner private services'
+  Type: AWS::EC2::SecurityGroup
+  Condition: CreateAppRunnerVPCEndpoint
+  Properties:
+    GroupDescription: !Join ['', [!Ref AppName, '-', !Ref EnvironmentName, AppRunnerVpcEndpointSecurityGroup]]
+    {{- if .VPCConfig.Imported}}
+    VpcId: {{.VPCConfig.Imported.ID}}
+    {{- else}}
+    VpcId: !Ref VPC
+    {{- end}}
+    Tags:
+      - Key: Name
+        Value: !Sub 'copilot-${AppName}-${EnvironmentName}-app-runner-vpc-endpoint'
```

Comment:
Would you mind checking the char limit for `GroupDescription` and `Tag` values.
I wonder if we should use go templates here and do something like:
```
{{truncate "copilot-.App-.Env-apprunner-vpc-endpoint" 255}}
```